modules/telco-troubleshooting-general-query-cluster.adoc

```diff
@@ -0,0 +1,78 @@
+clusteroperator.config.openshift.io/insights                                   4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/kube-apiserver                             4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/kube-controller-manager                    4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/kube-scheduler                             4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/kube-storage-version-migrator              4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/machine-api                                4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/machine-approver                           4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/machine-config                             4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/marketplace                                4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/monitoring                                 4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/network                                    4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/node-tuning                                4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/openshift-apiserver                        4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/openshift-controller-manager               4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/openshift-samples                          4.16.11   True        False         False      35d
+clusteroperator.config.openshift.io/operator-lifecycle-manager                 4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/operator-lifecycle-manager-catalog         4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/operator-lifecycle-manager-packageserver   4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/service-ca                                 4.16.11   True        False         False      62d
+clusteroperator.config.openshift.io/storage                                    4.16.11   True        False         False      62d
+----
+
+.Example output
+[source,terminal]
+----
+NAME                STATUS   ROLES                         AGE   VERSION
+node/ctrl-plane-0   Ready    control-plane,master,worker   62d   v1.29.7
+node/ctrl-plane-1   Ready    control-plane,master,worker   62d   v1.29.7
+node/ctrl-plane-2   Ready    control-plane,master,worker   62d   v1.29.7
+----
+--
```

Comment:
Consider adding a line to the end of this procedure to direct readers to the additional resources: `For more information about reviewing the information about your cluster, see "oc get" and "Reviewing pod status".`